### PR TITLE
Dw 3612

### PR DESCRIPTION
**Pull request by TheDogfather17**

### CODE REVIEW

File: acm.tf [hunk 1]
```diff
@@ -26,3 +26,39 @@ resource "aws_iam_policy" "pdm_dataset_acm" {
...
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${local.mgt_certificate_bucket}*",
+      "arn:aws:s3:::${local.env_certificate_bucket}/*",
+    ]
+  }
```
Comment (dan-hill2802) [inline acm.tf]: you can remove this statement, it is covered by the statement below

File: acm.tf [hunk 2]
```diff
@@ -26,3 +26,39 @@ resource "aws_iam_policy" "pdm_dataset_acm" {
...
+}
+
+resource "aws_iam_policy" "pdm_certificates" {
+  name        = "pdmTest"
```
Comment (dan-hill2802) [inline acm.tf]: looks like you forgot to update the name, maybe something like
```suggestion
  name        = "PdmGetCertificates"
```

File: data.tf [hunk 3]
```diff
@@ -28,13 +28,17 @@ data "aws_iam_policy_document" "pdm_dataset_generator_write_parquet" {
     effect = "Allow"
 
     actions = [
-      "s3:GetObject*",
+      "s3:Get*",
+      "s3:List*",
       "s3:DeleteObject*",
-      "s3:PutObject*",
+      "s3:Put*",
+
```
Comment (dan-hill2802) [inline data.tf]: shouldn't really have an empty line here

File: acm.tf (hunk 1 above)
Comment (TheDogfather17) [reply acm.tf]: removed

File: data.tf (hunk 3 above)
Comment (TheDogfather17) [reply data.tf]: removed

**Review by ChrisScottThomas — APPROVED** [review]
LGTM

**Pull request merged**